object-cache.php

```diff
@@ -918,6 +919,7 @@ function group_ops_stats( $op, $keys, $group, $size, $time, $comment = '' ) {
 
 		$keys = $this->strip_memcached_keys( $keys );
 
+		// @codeCoverageIgnoreStart
 		if ( $time > $this->slow_op_microseconds && 'get_multi' !== $op ) {
 			$this->increment_stat( 'slow-ops' );
 			$backtrace = null;
```

Comment:
It depends on the load on the CI servers whether this code path is triggered. We explicitly ignore this block for code coverage reporting in order not to upset CodeCov :-)